packages/create-cloudflare/templates/react/workers/ts/worker/index.ts

```diff
@@ -7,7 +7,5 @@ export default {
         name: "Cloudflare",
       });
     }
```

Comment:
Could we return a 404 instead?

```suggestion
    }
    
    return new Response(null, { status: 404 });
```